src/data-access/translations.ts

```diff
@@ -18,6 +18,13 @@ const getOne = async function(translationId: number) {
   return result;
 };
 
+const getStatusByWordByUser = async function(wordId: number, userId: number) {
```

Comment:
This is already available in `data-access/words.ts` as `getStatus`. Please use that one because it is an operation concerning words, not translations.